pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -115,15 +116,15 @@ func (configgen *ConfigGeneratorImpl) buildOutboundClusters(env *model.Environme
 			if config != nil {
 				destinationRule := config.Spec.(*networking.DestinationRule)
 				convertIstioMutual(destinationRule, upstreamServiceAccounts)
-				applyTrafficPolicy(defaultCluster, destinationRule.TrafficPolicy, port)
+				applyTrafficPolicy(defaultCluster, destinationRule.TrafficPolicy, port, env.Mesh.SdsUdsPath)
```

Comment:
just pass env.. otherwise the arg list keeps growing

Reply:
updated.